### ISSUE TITLE
Fix i386 build on Ubuntu 22.04

### ISSUE THREAD

**Issue by disconnect3d** [pwndbg/pwndbg#1714]
The Ubuntu 22.04 build on CI is failing on the `test_largebins_size_range_32bit_big` and `test_smallbins_sizes_32bit_big` tests.

This can be seen here https://github.com/pwndbg/pwndbg/actions/runs/4973800358/jobs/8899872381#step:6:538 or here:

```
        for bin_index, size_range in enumerate(command_output):
>           assert size_range.split(":")[0] == expected[bin_index]
E           AssertionError: assert 'This might not work in all cases. Use `help set resolve-heap-via-heuristic` for more details.' == '0x3f0-0x3f0'
E             - 0x3f0-0x3f0
E             + This might not work in all cases. Use `help set resolve-heap-via-heuristic` for more details.
```

I believe this issue comes from the lack of i386 debugging symbols since its package stopped to be installed?:
```
+ sudo apt-get install -y libc6-dbg:i386
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libc6:i386 : Depends: libgcc-s1:i386 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by zuiderkwast** [comment]
I had the same problem in an unrelated repo. Explicitly adding the package `libgcc-s1:i386` to the `apt-get install` command line solved the issue.

I don't know if this is a temporary glitch in the Ubuntu package repo that will resolve itself automatically (or by the Ubuntu maintainers).

**Comment by Jj0YzL5nvJ** [comment]
Another unrelated repository here, in my case it's fixed by explicitly adding `libatomic1:i386`, `libgcc-s1:i386` and `libstdc++6:i386`.

I know that Ubuntu 22.04 usually retains upgrades of some repositories, but I don't know the exact conditions.
When it happens I usually use aptitude instead of apt... but you have to watch out it so it doesn't cause other havoc by "fixing dependencies" ...it's not safe to automate it.
Adding said repositories as a workaround is much safer.

**Comment by gsingh93** [comment]
Would appreciate a PR from those of you that were able to fix the issue locally or in other repos 🙂

**Comment by gsingh93** [comment]
Fixed in #1716
